api/src/api/members.js

```diff
@@ -72,6 +72,54 @@ router.post(
   }),
 );
 
+// Gives the enum options to the frontend to populate the dropdown.
+// The key in the 'options' object must be the same as the DB attribute.
+router.get(
+  '/options',
+  requireRegistered,
+  errorWrap(async (req, res) => {
+    const options = {};
+    Member.schema.eachPath((pathname, schemaType) => {
+      const { enum: optionsEnum } = schemaType.options;
+      if (optionsEnum) {
+        options[pathname] = optionsEnum.map(
+          (option) => new Object({ label: option, value: option }),
```

Comment:
Very minor nitpick :P but this can also be written as
`(option) => ({ label: option, value: option })`

If directly returning an object in lambda it must be wrapped with parentheses.

Reply:
I was wondering how to do that :)